frontends/mit-open/src/components/FieldAvatar/FieldAvatar.tsx

```diff
@@ -61,18 +61,21 @@ const AvatarContainer = styled.div<AvatarStyleProps>`
   align-items: center;
   height: ${({ imageSize }) => IMG_SIZES[imageSize]};
   width: auto;
+  ${({ theme }) => theme.breakpoints.down("sm")} {
+    height: auto;
+    width: 100%;
+  }
 `
 const AvatarImg = styled.img<AvatarStyleProps>`
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: center;
   min-height: 0;
   min-width: 0;
   ${({ imageVariant }) =>
     imageVariant === "inverted" ? "filter: saturate(0%) invert(100%);" : ""}
   height: ${({ imageSize }) => IMG_SIZES[imageSize]};
   width: auto;
+  ${({ theme }) => theme.breakpoints.down("sm")} {
+    height: min(calc(100vh), ${({ imageSize }) => IMG_SIZES[imageSize]});
+  }
```

Comment:
IMO you should remove this. It's basically `min(100vh, X)` where X = 22px, 57px, or 90px.  Unless the screen is smaller than 22px, 57px, or 90px tall, this won't do anything.